app/src/main/java/com/wpi/audiojournal/models/Category.kt

```diff
@@ -0,0 +1,6 @@
+package com.wpi.audiojournal.models
+
+class Category(
```

Comment:
I think class works for models, but you can also use data class (Classname) for models if needed

Reply:
Yeah I know